lib/caret.controller.js

```diff
@@ -165,6 +191,13 @@ leodido.controller.Caret = function($scope, $element, $attrs) {
       };
       setPositionPolyfill = function() {
       };
+      setSelectionTextPolyfill = function(text) {
+        // Browser not supported - set at the end.
+        target.value += text;
+
+        // Scroll to the end of textarea to show inserted.
+        target.scrollTop = 100000;
+      };
```

Comment:
IMHO, It's not an expected behaviour. 
I understand there's no way to replace the text selection in this case... but just doing nothing could be better than an opinionated approach.